src/mixins/transparentize.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import { isRgb, isRgba, isHsl, isHsla, isHex } from '../internalHelpers/_isValidColor'
+import hexToRgb from '../internalHelpers/_hexToRgb'
+
+const getNewTransparency = (currentTransparency: number, percentage: number) : number => (
+  currentTransparency * (1 - (percentage / 100))
+)
+
+const getColorValuesInBrackets = (color: string) : string => (
+  color.slice(color.indexOf('(') + 1, color.indexOf(')'))
+)
+
+const getValuesTillLastComma = (color: string) : string => (
+  color.slice(0, color.lastIndexOf(','))
+)
+
+const transparentizeRgb = (rgb: string, percentage: number) : string => (
+  `rgba(${getColorValuesInBrackets(rgb)}, ${getNewTransparency(1, percentage)})`
+)
+
+const getLastColorValue = (color: string) : number => (
+  Number(
+    color.slice(
+      color.lastIndexOf(',') + 1,
+      color.indexOf(')'),
+    ),
+  )
+)
+
+function transparentize(color: string, percentage : number) {
```

Comment:
Can you make sure it accepts a range from 0 to 1 instead of 0 to 100.

Why?
I believe it's easier to comprehend for people that if the current value is 0.8 that by passing 0.2 it would end up with 0.6. Having an 0.8 alpha and passing in 20 to reduce it to 0.6 feels strange to me. Thats also the reason why saturation, lightness is using 0 to 1 and we should stay consistent across the lib.

Can you also remove the space after percentage.